app/services/idv/flows/inherited_proofing_flow.rb

```diff
@@ -5,7 +5,6 @@ class InheritedProofingFlow < Flow::BaseFlow
         get_started: Idv::Steps::InheritedProofing::GetStartedStep,
         agreement: Idv::Steps::InheritedProofing::AgreementStep,
         verify_info: Idv::Steps::InheritedProofing::VerifyInfoStep,
-        verify_phone: Idv::Steps::InheritedProofing::PhoneStep,
```

Comment:
If the class is now unused, can we remove the class file as well? 

https://github.com/18F/identity-idp/blob/main/app/services/idv/steps/inherited_proofing/phone_step.rb

Reply:
@aduth thank you for pointing that out. Removed the file now.